### PR TITLE
add new data into table 'tags', 'habits-tags', 'tag-translations'

### PR DESCRIPTION
**Pull request by LiliaMokhnatska**
# Green City add new data into data base   PR

## Summary Of Issue :
Need to add new tags in table 'tag' and 'tag-translation' in database and relate it with  habits (habits-tags'') according to:
https://docs.google.com/spreadsheets/d/1IY4CJlBLb1AnoP2BcFlgzTHQp3bAb3Y0/edit#gid=1314547193

## Issue Link :
https://github.com/ita-social-projects/GreenCity/issues/5065

## Summary Of Changes :

1. changed number of symbols in column 'name' in table 'tag-translation' with changelog

2. add new data into table 'tags', 'habits-tags', 'tag-translations'

## Code reviewers

- [ ] @github_username

## Added
add new data into table 'tags', 'habits-tags', 'tag-translations'

## Changed 
changed number of symbols in column 'name' in table 'tag-translation' with changelog

## How to test :
1. User need to be logged in http://localhost:8060/swagger-ui.html#/own-security-controller
2. Register Token
4. Send 'GET' request tags/v2/search  choose type habit 
image

image

## CHECK LIST
- [x] Code is up-to-date with the `dev` branch.
- [ ] You've successfully built and run the tests locally.
- [ ] There are new or updated unit tests validating the change.
- [x] JIRA/ Github Issue number & title in PR title (#5065: Ticket title)
- [x] This template filled (above this section).
- [x] Sonar's report does not contain bugs, vulnerabilities, security issues, code smells ar duplication
- [x] `NEED_REVIEW` and `READY_FOR_REVIEW` labels are added.
- [x] All files reviewed before sending to reviewers

### CODE REVIEW

**Review by ABbondar — APPROVED** [review]

File: dao/src/main/resources/db/changelog/db.changelog-master.xml [hunk 1]
```diff
@@ -133,5 +133,12 @@
     <include file="db/changelog/logs/ch-update-into-tag-translations-Hlazova.xml"/>
     <include file="db/changelog/logs/ch-add-column-parentComment-Vatuliak.xml"/>
     <include file="db/changelog/logs/ch-change-eco-news-create-user-dislikes.xml"/>
+    <include file="db/changelog/logs/ch-insert-into-tags-Mokhnatska.xml"/>
+    <include file="db/changelog/logs/ch-change-tag-translations-column-name-Mokhnatska.xml"/>
+    <include file="db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml"/>
+
+    <include file="db/changelog/logs/ch-insert-into-habits-tags-Mokhnatska.xml"/>
+
+
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/db.changelog-master.xml]: please remove these blank lines

Comment (LiliaMokhnatska) [reply dao/src/main/resources/db/changelog/db.changelog-master.xml]: ok

File: dao/src/main/resources/db/changelog/logs/ch-insert-into-tags-Mokhnatska.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet id="AddNewTags" author="Lilia Mokhnatska">
+        <insert tableName="tags">
+            <column name="id">20</column>
+            <column name="type">HABIT</column>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-insert-into-tags-Mokhnatska.xml:8]: Do we have other types?
I see you add only `HABIT` type

File: dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">71</column>
+            <column name="name">Recycling / Waste sorting</column>
+            <column name="tag_id">24</column>
+            <column name="language_id">2</column>
+        </insert>
+        <insert tableName="tag_translations">
+            <column name="id">72</column>
+            <column name="name">Переделать/сортировать отходы</column>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml]: do we really need russian translation?

File: dao/src/main/resources/db/changelog/logs/ch-insert-into-tags-Mokhnatska.xml (hunk 2 above)
Comment (LiliaMokhnatska) [reply dao/src/main/resources/db/changelog/logs/ch-insert-into-tags-Mokhnatska.xml:8]: image
yes this table has tags for Event, Places та Habits

File: dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml (hunk 3 above)
Comment (RomaMocherniuk) [reply dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml]: I do not see the russian  version of the site.
image

Comment (RomaMocherniuk) [reply dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml]: we should not support russian
#russiaisaterroriststate

File: dao/src/main/resources/db/changelog/logs/ch-change-tag-translations-column-name-Mokhnatska.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet id="modifyNumberOfCharacters" author="Lilia Mokhnatska">
+        <modifyDataType tableName="tag_translations" columnName="name" newDataType="varchar(30)"/>
+    </changeSet>
+</databaseChangeLog>
+
+
+
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/logs/ch-change-tag-translations-column-name-Mokhnatska.xml]: please remove blank lines

File: dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml (hunk 3 above)
Comment (LiliaMokhnatska) [reply dao/src/main/resources/db/changelog/logs/ch-insert-into-tag-translations-Mokhnatska.xml]: I'll remove it

File: dao/src/main/resources/db/changelog/logs/ch-change-tag-translations-column-name-Mokhnatska.xml (hunk 4 above)
Comment (LiliaMokhnatska) [reply dao/src/main/resources/db/changelog/logs/ch-change-tag-translations-column-name-Mokhnatska.xml]: ok

**Review by RomaMocherniuk — APPROVED** [review]

**Review by JJJazl — APPROVED** [review]

**Review by juseti — APPROVED** [review]

**Review by jpgcz — APPROVED** [review]

**Pull request merged**